Sprint-2/implement/vat.js

```diff
@@ -8,3 +8,10 @@
 // Given a number,
 // When I call this function with a number
 // it returns the new price with VAT added on
+
+function price_with_VAT(price){
```

Comment:
If price should be in £, could you rename the parameter `price` to make that clearer to someone who might use your function?

Reply:
updated